src/main/g8/src/main/resources/application.conf

```diff
@@ -0,0 +1,6 @@
+akka {
+  # logging configuration controlling logging through classic actor APIs
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
```

Comment:
Strange if this would be needed. We automatically activate that logger when akka-actor-typed is used, see https://github.com/akka/akka/blob/1b36a3ee7d37a0a8dc257bc5195dc843e73b7cb9/akka-actor-typed/src/main/resources/reference.conf#L54-L66

Reply:
Oh, I had forgotten about that magic. 

Maybe something isn't working as it should then. I retried just in case I missed something but I can reproduce:
1. create a new project from the g8 template
2. change logger.xml to debug for the root logger
3. sbt run, no debug (or info) logs printed from Akka, only info from the app actors

Reply:
`akka.loglevel = DEBUG` is missing, but above loggers is not needed. This is what I see when I change logback.xml and add application.conf with DEBUG only.

```
[info] [2023-05-02 16:33:32,577] [INFO] [akka.event.slf4j.Slf4jLogger] [AkkaQuickStart-akka.actor.default-dispatcher-3] [] - Slf4jLogger started
[info] [2023-05-02 16:33:32,581] [DEBUG] [akka.event.EventStream] [AkkaQuickStart-akka.actor.default-dispatcher-3] [EventStream(akka://AkkaQuickStart)] - logger log1-Slf4jLogger started
[info] [2023-05-02 16:33:32,582] [DEBUG] [akka.event.EventStream] [AkkaQuickStart-akka.actor.default-dispatcher-3] [EventStream(akka://AkkaQuickStart)] - Default Loggers started
[info] [2023-05-02 16:33:32,607] [DEBUG] [akka.serialization.Serialization(akka://AkkaQuickStart)] [AkkaQuickStart-akka.actor.default-dispatcher-3] [akka.serialization.Serialization(akka://AkkaQuickStart)] - Replacing JavaSerializer with DisabledJavaSerializer, due to `akka.actor.allow-java-serialization = off`.
[info] [2023-05-02 16:33:32,644] [INFO] [com.example.Greeter$] [AkkaQuickStart-akka.actor.default-dispatcher-5] [akka://AkkaQuickStart/user/greeter] - Hello Charles!
[info] [2023-05-02 16:33:32,644] [INFO] [com.example.GreeterBot$] [AkkaQuickStart-akka.actor.default-dispatcher-5] [akka://AkkaQuickStart/user/Charles] - Greeting 1 for Charles
[info] [2023-05-02 16:33:32,644] [INFO] [com.example.Greeter$] [AkkaQuickStart-akka.actor.default-dispatcher-5] [akka://AkkaQuickStart/user/greeter] - Hello Charles!
[info] [2023-05-02 16:33:32,644] [INFO] [com.example.GreeterBot$] [AkkaQuickStart-akka.actor.default-dispatcher-6] [akka://AkkaQuickStart/user/Charles] - Greeting 2 for Charles
[info] [2023-05-02 16:33:32,644] [INFO] [com.example.Greeter$] [AkkaQuickStart-akka.actor.default-dispatcher-6] [akka://AkkaQuickStart/user/greeter] - Hello Charles!
[info] [2023-05-02 16:33:32,644] [INFO] [com.example.GreeterBot$] [AkkaQuickStart-akka.actor.default-dispatcher-6] [akka://AkkaQuickStart/user/Charles] - Greeting 3 for Charles
```

Reply:
Alright, the surprising part is that we don't let debug trough to the underlying logger when auto-hooking it up I guess.

Reply:
We don't amend the loglevel automatically, only the loggers. Default is INFO.